worldedit-core/src/main/java/com/sk89q/worldedit/internal/util/BiomeMath.java

```diff
@@ -19,53 +19,54 @@
 
 package com.sk89q.worldedit.internal.util;
 
-import net.royawesome.jlibnoise.MathHelper;
-
 public class BiomeMath {
 
-    // From BiomeArray / BiomeContainer
-    public static final int HORIZONTAL_SECTION_COUNT = (int) Math.round(Math.log(16.0D) / Math.log(2.0D)) - 2;
-    public static final int VERTICAL_SECTION_COUNT = (int) Math.round(Math.log(256.0D) / Math.log(2.0D)) - 2;
-    public static final int HORIZONTAL_BIT_MASK = (1 << HORIZONTAL_SECTION_COUNT) - 1;
-    public static final int VERTICAL_BIT_MASK = (1 << VERTICAL_SECTION_COUNT) - 1;
-
-    private BiomeMath() {
-    }
-
-    /**
-     * Compute the index into the MC biome array, for non-extended-height worlds.
-     *
-     * @param x the block x coordinate
-     * @param y the block y coordinate
-     * @param z the block z coordinate
-     * @return the index into the standard MC biome array
-     */
-    public static int computeBiomeIndex(int x, int y, int z) {
```

Comment:
Removing these, even if they are in the internal package is a major upstream compatibility breakage. All the public methods should just return 0 .

Reply:
it's in the internal package so should be considered very much not API.

Reply:
I'll mark this as resolved but I hope it doesn't come to bite us in the butt.